src/tests/src/solver/simulation/remix-storage/tests-multi-storage-remix.cpp

```diff
@@ -0,0 +1,266 @@
+#define BOOST_TEST_MODULE multi storage remix
+
+#define WIN32_LEAN_AND_MEAN
+
+#include <unit_test_utils.h>
+#include <vector>
+
+#include <boost/test/unit_test.hpp>
+
+#include "antares/solver/simulation/remix-storage/create-storage-for-remix.h"
+#include "antares/solver/simulation/remix-storage/remix-utils.h"
+#include "antares/solver/simulation/remix-storage/shave-peaks-by-remix-storage-gen.h"
+
+using namespace Antares::Solver::Simulation;
+
+template<unsigned int size>
+struct STS_setup
+{
+    STS_setup()
+    {
+        withdrawal.assign(size, 0);
+        injection.assign(size, 0);
+        levels.assign(size, 0.);
+        pmax.assign(size, std::numeric_limits<double>::max());
+        inflows.assign(size, 0.);
+        ovf.assign(size, 0.);
+        lowRuleCurve.assign(size, 0.);
+        upRuleCurve.assign(size, capacity);
+    }
```

Comment:
Initialize members with their constructors
```suggestion
    STS_setup(): withdrawal(size, 0),
    injection(size, 0),
    // etc.
    {
    }
```

Reply:
Done